manifests/peer-validation.yaml

```diff
@@ -0,0 +1,152 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kilo
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: "peer-validation.kilo.svc"
```

Comment:
I think it's a bit funny to name the webhook as a DNS name. Normally these resources have names like `kilo-webhook` or something